app/Http/Controllers/Company/JobController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Controllers\company;
+
+use App\Http\Controllers\Controller;
+use App\Models\Job;
+use App\Models\User;
+use http\Client\Response;
+use Illuminate\Http\Request;
+
+class JobController extends Controller
+{
+public function createJob(Request $request ,$user_id){
+    $company=User::find($user_id);
+    if(!$company){
+        return Response()->json([
+            'message'=>'no company'],404);
+    }
+    $job =$company->jobs()->create([
```

Comment:
why you use this ? ... we can Do that
`
$job = Job::create([
'user_id' => $company->id 
 'sub_category_id => $request['sub_category_id'] 
 ...  
`